lib/addons/index.ts

```diff
@@ -4,6 +4,11 @@ export { AwsLoadBalancerControllerAddOn} from './aws-loadbalancer-controller'
 export { CalicoAddOn } from './calico'
 export { ContainerInsightsAddOn } from './container-insights'
 export { ClusterAutoScalerAddOn } from './cluster-autoscaler'
+export { ExternalDnsAddon, LookupHostedZoneProvider, DelegatingHostedZoneProvider } from './external-dns'
 export { MetricsServerAddOn } from './metrics-server'
 export { NginxAddOn } from './nginx'
-export { XrayAddOn } from './xray'
\ No newline at end of file
+export { XrayAddOn } from './xray'
+
+export class Constants {
```

Comment:
Should we consider a separate `constants.ts` file for versions and enums?

Reply:
I added constants for just add-ons at the moment. I suppose those could be different for each category - addons, cluster providers, teams, but I don't exclude we may require a global one in the future as well. Assuming the approach of constants per module is viable, unless we have a case when it does not work.